OrbitService/Utils.cpp

```diff
@@ -176,17 +176,16 @@ static std::optional<pid_t> ProcEntryToPid(const std::filesystem::directory_entr
 }
 
 std::vector<pid_t> GetAllPids() {
-  const auto emplace_back_if_has_value = [](auto vec, const auto& val) {
-    if (val.has_value()) {
-      vec.emplace_back(std::move(val.value()));
-    }
+  fs::directory_iterator proc{"/proc"};
+  std::vector<pid_t> pids;
 
-    return std::move(vec);
-  };
+  for (const auto& entry : proc) {
+    if (auto pid = ProcEntryToPid(entry); pid) {
```

Comment:
As you know well by now, I liked the use of `has_value` in the original code! 😄

Reply:
Sure. Fine with me. :+1: I added the `has_value`. Thanks!